src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 All of the encoders you list are "unknown". only ```h264_nvenc``` is working. I guess the nightly build isn't compiled with the correct options? 

I'd like to estabilish something once and for all: I've been burned by AMD too many times (that is, every time I've fallen to the temptation of their cheaper prices) to ever trust them again. Their quality simply isn't up to par, especially when it comes to drivers. I used to like ATI GPU's many years ago, but as they were "AMD'ed" the quality problems started becoming evident for the GPU's as well. I suffered with the Radeon 4770 HD for a while, but the fan noise combined with the frequent crashes and mediocre perfocmance ultimately made me have to bite the bullet and buy a GeForce again. Intel isn't making serious GPUs this far, so they don't count, which means you'll only see Nvidia GPUs here just like you'll only see Intel CPUs 😉